### PR TITLE
Adding sample of Find-and-Replace-text-with-content-control

### PR DESCRIPTION
**Pull request by sharonaumao**
Added KB documentation sample. The sample is:

- Replace-text-with-content-control

### CODE REVIEW

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    //Append a picture to the cell.
+                    WPicture picture = cellPara.AppendPicture(imageStream) as WPicture;
+                    picture.Height = 88.3f;
+                    picture.Width = 142.2f;
+                    //Add a new paragraph to the next cell.
+                    cellPara = row.Cells[1].AddParagraph();
+                    row.Cells[1].CellFormat.VerticalAlignment = VerticalAlignment.Top;
+                    cellPara.ParagraphFormat.BeforeSpacing = 12f;
+                    //Append text to the cell.
+                    IWTextRange txt = cellPara.AppendText("Mountain-200");
```
Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:38]: @sharonaumao - Please use proper variable name `txt`

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    cellPara.ParagraphFormat.AfterSpacing = 3f;
+                    //Set the text format.
+                    txt.CharacterFormat.Bold = true;
+                    txt.CharacterFormat.FontName = "Arial";
+                    txt.CharacterFormat.FontSize = 16f;
+                    //Add a new paragraph.
+                    cellPara = row.Cells[1].AddParagraph();
+                    //Append texts to the paragraph of a cell.
+                    txt = cellPara.AppendText("Product No: BK-M68B-38");
```
Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:47]: @sharonaumao - Do we need to assign all the text ranges in a `txt` variable?

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                    //Add block content control into the Word document.
+                    BlockContentControl blockContentControl = new BlockContentControl(document, ContentControlType.RichText);
+                    //Add a new table to the block content control.
+                    WTable table = blockContentControl.TextBody.AddTable() as WTable;
+                    //Specify the total number of rows and columns.
+                    table.ResetCells(1, 2);
+                    //Get first row of the table.
+                    WTableRow row = table.Rows[0];
+                    //Add a new paragraph to the first cell of the table.
+                    IWParagraph cellPara = row.Cells[0].AddParagraph();
```
Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:26]: @sharonaumao - Please use proper variable name `cellPara`

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Replace_text_with_content_control
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```
Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:12]: @sharonaumao - Please use common variable name for input file stream in every KB and UG topics. Please refer your previous samples.

Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:12]: @sharonaumao - Please provide proper `FileAccess` and `FileShare` permission.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs (hunk 1 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs]: @Ramaraj-Marimuthu -I have changed

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs (hunk 4 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs]: @Ramaraj-Marimuthu -It is done

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                    IWTextRange text = cellParagraph.AppendText("Mountain-200");
+                    cellParagraph.ParagraphFormat.AfterSpacing = 3f;
+                    //Set the text format.
+                    text.CharacterFormat.Bold = true;
+                    text.CharacterFormat.FontName = "Arial";
+                    text.CharacterFormat.FontSize = 16f;
+                    //Add a new paragraph.
+                    cellParagraph = row.Cells[1].AddParagraph();
+                    //Append texts to the paragraph of a cell.
+                    text = cellParagraph.AppendText("Product No: BK-M68B-38\nSize: 38\nWeight: 25\nPrice: $2,294.99\n\n");
```
Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:47]: @sharonaumao - What is the purpose of assigning a appended textrange in  `text` variable here?

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs (hunk 5 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs]: @Ramaraj-Marimuthu -I have appended it directly now

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+                    text.CharacterFormat.FontName = "Arial";
+                    text.CharacterFormat.FontSize = 16f;
+                    //Add a new paragraph.
+                    cellParagraph = row.Cells[1].AddParagraph();
+                    //Append texts to the paragraph of a cell.
+                    cellParagraph.AppendText("Product No: BK-M68B-38\nSize: 38\nWeight: 25\nPrice: $2,294.99\n\n");
+                    //Specify the table borders border type.
+                    table.TableFormat.Borders.BorderType = BorderStyle.Single;
+
+                    TextBodyPart textBodyPart = new TextBodyPart(document);
```
Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:51]: @sharonaumao - Please add a XML comments for these codes.

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream inputStream = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Load an existing Word document.
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
+                {
+                    //Add block content control into the Word document.
```
Comment (Ramaraj-Marimuthu) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:17]: @sharonaumao - Please modify the XML comments as we discussed.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs (hunk 6 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:51]: @Ramaraj-Marimuthu -I have added

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs (hunk 7 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs]: @Ramaraj-Marimuthu -I have done so Sir

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have any concerns in this code changes.

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream inputStream = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Load an existing Word document.
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
+                {
+                    //Create a BlockContentControl object.
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:17]: //Create a BlockContentControl.

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+                    IWTextRange text = cellParagraph.AppendText("Mountain-200");
+                    cellParagraph.ParagraphFormat.AfterSpacing = 3f;
+                    //Set the text format.
+                    text.CharacterFormat.Bold = true;
+                    text.CharacterFormat.FontName = "Arial";
+                    text.CharacterFormat.FontSize = 16f;
+                    //Add a new paragraph.
+                    cellParagraph = row.Cells[1].AddParagraph();
+                    //Append texts to the paragraph of a cell.
+                    cellParagraph.AppendText("Product No: BK-M68B-38\nSize: 38\nWeight: 25\nPrice: $2,294.99\n\n");
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:47]: //Append text to the paragraph of a cell.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs (hunk 8 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs]: @MohanaselvamJothi -I have changed the comment

File: Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs (hunk 9 above)
Comment (sharonaumao) [reply Find-and-Replace/Replace-text-with-content-control/.NET-Standard/Replace-text-with-content-control/Program.cs:47]: @MohanaselvamJothi -I have changed the comment

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**